Cannot create new project (Ubuntu 19.04)

When running the command `npx create-cljs-app my-app`, the following error is returned:

```
npx: installed 24 in 1.633s
Cannot find module '../dist/lib'
Require stack:
- /home/iwsfutcmd/.npm/_npx/32025/lib/node_modules/create-cljs-app/bin/create-cljs-app.js
```

Similar errors are returned when using `npm init` and `yarn create`.

Currently running this on Ubuntu 19.04, Node v12.8.0

--- a/.github/workflows/nodejs.yml
+++ b/.github/workflows/nodejs.yml
@@ -35,9 +35,15 @@ jobs:
       with:
         node-version: '12.x'
     - name: Install
-      run: yarn    
+      run: yarn
+    - name: Build
+      run: yarn build      
     - name: Semantic Release
       uses: cycjimmy/semantic-release-action@v2
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        NPM_TOKEN: ${{ secrets.NPM_TOKEN }}      
+        NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+    - name: Wait for release to propagate
+      run: sleep 10
+    - name: Try to use release
+      run: yarn create cljs-app my-cljs-app

